bigbluebutton-html5/imports/ui/services/features/index.js

```diff
@@ -80,6 +80,10 @@ export function isPresentationEnabled() {
   return getDisabledFeatures().indexOf('presentation') === -1;
 }
 
+export function sendInvitationToIncludedModerators() {
+  return getDisabledFeatures().indexOf('sendInvitationToIncludedModerators') === -1;
+}
+
```

Comment:
```suggestion
```

I think it is not necessary to have a `disabledFeatures` for this option once it is just a minor feature.